plugins/content/entry_permission_level/lib/api/filters/KalturaPermissionLevelUserEntryFilter.php

```diff
@@ -1,14 +1,64 @@
-<?php
-/**
- * @package plugins.entryPermissionLevel
- * @subpackage api.filters
- */
-class KalturaPermissionLevelUserEntryFilter extends KalturaUserEntryFilter
-{
-	public function getListResponse(KalturaFilterPager $pager, KalturaDetachedResponseProfile $responseProfile = null)
-	{
-		$this->typeEqual = EntryPermissionLevelPlugin::getApiValue(PermissionLevelUserEntryType::PERMISSION_LEVEL);
-		$response = parent::getListResponse($pager, $responseProfile);
-		return $response;
-	}
-}
+<?php
+/**
+ * @package plugins.entryPermissionLevel
+ * @subpackage api.filters
+ */
+class KalturaPermissionLevelUserEntryFilter extends KalturaUserEntryFilter
+{
+	/**
+	 * @var KalturaPermissionLevelArray
+	 */
+	public $permissionLevels;
+
+	/**
+	 * @var int
+	 * @readonly
+	 */
+	public $permissionLevelsBitmask;
```

Comment:
2 comments:
1. Why id permissionLevelsBitmask public or even exposed in the API? It should be private an internal only. The only thing the client should send is the array and we should calc the bitmask internally only.
2. I don't get why isn't the filters permissionLevels exposed in the map_between_objects? how does it even work?